executor/kava/guide.md

```diff
@@ -0,0 +1,221 @@
+# Set up
+## Kava 
+1. Clone kava [repo]( https://github.com/Kava-Labs/kava)
+2. Checkout branch `develop`
+3. Install the `kvd` and `kvcli` binaries with `make install`
+4. Initialize the blockchain with chain-id `testing`
+5. Create 2 accounts `deputy` and `user` that both have a bnb balance of 100000000000.
+6. In the genesis file located in `/contrib/testnet-5000/genesis-examples/genesis-bep3.json`, enter `deputy` address in the `deputy_address` field under bep3 params.
+
+## Bnbchain
+### Option 1: use the existing cloud server
+We've set up a cloud server containing `bnbchaind` with the expected accounts/mnemonics and IP address. It can be used for testing, contact system admin @karzak for ssh access.
+```bash
+ssh ubuntu@ec2-3-231-211-245.compute-1.amazonaws.com
+```
+
+### Option 2: manual set up
+1. Set up _local_ bnbchain with the steps provided [here](https://docs.binance.org/fullnode.html) using flag `--chain-id Binance-Chain-Tigris`.
+**Note**: do not join the bnbchain mainnet. You must set up a local chain with chain id "Binance-Chain-Tigris".
+2. With `bnbchaind` and `bnbcli` installed, create 2 accounts `deputy` and `user` that both have a BNB balance of 10000000000000.
+
+## bep3-deputy
+1. Clone this repo
+2. Checkout branch `kava-deputy`
+3. A sample config file is located at `/config/config.json`. Sections `bnb_config` and `kava_config` must be modified to match your bnbchain and kava chain variables. Enter kava `deputy` address and mnemonic and bnbchain `deputy` address and mnemonic. The `rpc_addr` field may also need to be updated.
+4. Build bep3-deputy with `make build`
+
+# Start processes
+## Kava
+Start Kava
+```bash
+kvd unsafe-reset-all
```

Comment:
Add `kvcli config chain-id testing` to avoid chain-id issues

Reply:
https://github.com/Kava-Labs/bep3-deputy/pull/1/commits/badeb0de0be59aea80df80664c5f2efce6b796fa

Reply:
https://github.com/Kava-Labs/bep3-deputy/pull/1/commits/badeb0de0be59aea80df80664c5f2efce6b796fa